pkg/nfd-master/nfd-master.go

```diff
@@ -284,6 +295,41 @@ func (m *nfdMaster) Run() error {
 	}
 }
 
+func (m *nfdMaster) processNodeLabelRequest() bool {
+	nodeName, quit := m.queue.Get()
+	if quit {
+		return false
+	}
+	defer m.queue.Done(nodeName)
+	err := m.nfdAPIUpdateOneNode(nodeName.(string))
+
+	if err != nil {
```

Comment:
you can inline err definition:
```suggestion
	if err := m.nfdAPIUpdateOneNode(nodeName.(string)); err != nil {
```